### DISCUSSION TITLE
How to add title tag to head on the client side?

### DISCUSSION THREAD

**Discussion by MincePie** [vercel/next.js#27283] [Help]
I'm trying to follow the instructions to add an env.variable to the head tag of my typescript app.

In my env file, I have defined a variable with the NEXT_PUBLIC prefix and then in the title tag I use it as shown in the docs - as follows:

        <title>`process.env.NEXT_PUBLIC_REACT_APP_TITLE`</title>

The result is that these word, including the backticks are printed in the browser tab. If I remove the backticks from my index.tsx file, the update gets printed in the browser tab in the same way.

When I use this in the pages/index  - the above is the result.

When I move it to the _document.js file, I get the actual env variable to load in the browser tab on the home page, but it reverts to 'localhost:2000/about' when I follow a link to any other page. I also get a warning in the terminal that says:

Warning: <title> should not be used in _document.js's <Head>. https://nextjs.org/docs/messages/no-document-title

When I follow the link in this warning, I get a suggestion to define it in _app.tsx instead. When I do that, I'm back to the original problem - the tab in the browser prints process.env...

**Comment by minminminmmm** [comment]
Try it like this <title>{process.env.NEXT_PUBLIC_REACT_APP_TITLE}</title>

**Reply by MincePie** [reply]
thank you
